README.md

```diff
@@ -42,6 +42,18 @@ Some crates of the **matrix-rust-sdk** can be embedded inside other
 environments, like Swift, Kotlin, JavaScript, Node.js etc. Please,
 explore the [`bindings/`](./bindings/) directory to learn more.
 
+## Testing
+
+You can run most of the tests that also happen in CI also using `cargo xtask ci`.
+This needs a few dependencies to be installed, as it also runs automatic WASM tests.
```

Comment:
Can you start a list of "at least these dependencies" that we can expand later? Things that I know off the top of my head are `nextest`, `uniffi_bindgen`, `typos` and `wasm-pack`.

Reply:
Hm, maybe `xtask ci bindings` isn't included in `xtask ci`. We can leave it out for now.